assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -37,6 +37,33 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 		} );
 	},
 
+	updateBackgroundImagesSizes( control, values ) {
+		if ( control.name.includes( 'background_image' ) ) {
+			const imageSize = values.background_dimension || false;
```

Comment:
why do you need to make it false? it's already falsy at that point

Reply:
Removed.

Reply:
Where is the removal?